File: recursion/n_times_hello.cpp
#include<bits/stdc++.h>
using namespace std;
void printHello(int n){
    if(n<=0)                //base case
    return;   
    cout<< "Hello\n";              //task
    printHello(n-1);        //recursive call for the smaller problem
}
int main(){
    int N;
    cout << "Enter N : ";
    cin >> N;
    printHello(N);
    return 0;
}

File: recursion/n_times_hello.java
import java.util.Scanner;
public class n_times_hello{
    static void printHello(int n){
        if(n<=0)        //base case
        return; 
        System.out.println("Hello");
        printHello(n-1);     // recursive call
    }
    public static void main(String[]args){
        System.out.println("Enter N");
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        printHello(N);
        sc.close();
    }
}